spec/wings/valkyrie/persister_spec.rb

```diff
@@ -415,7 +415,6 @@ class Custom < ActiveFedora::Base
     # internal_resource="Wings::ActiveFedoraConverter::DefaultWork"
     # so the CustomResource defined above will not be persisted as such.
     it "can find that resource again" do
-      Wings::ModelRegistry.register(resource_class, Custom)
```

Comment:
Definitely like removing this from a spec.  There's an implication that there is a conceptual "wrap registration" for this context.

```ruby
Wings::ModelRegistry.with_temporary_registrations([resource_class, Custom]) do
  # spec logic that depends on the registration
end
# and here the temporary registrations are gone